vars.development.yml

```diff
@@ -10,7 +10,7 @@ CKAN_API_URL: https://catalog-next-dev-admin-datagov.app.cloud.gov
 
 HARVEST_RUNNER_APP_GUID: 40dba74b-4916-447e-914b-77525de5e608
 
-MDTRANSLATOR_URL: https://mdtranslator.app.cloud.gov/translates
+MDTRANSLATOR_URL: http://mdtranslator-development.apps.internal:8080/translates
```

Comment:
Do you know if this affects how we invoke the service?

```
mdt_url = os.getenv("MDTRANSLATOR_URL")
resp = requests.post(mdt_url, json=data)
```
https://github.com/GSA/datagov-harvester/blob/parellize-ckan-network-calls/harvester/harvest.py#L798-L799

Reply:
Requests transparently handles the URL with port apparently, because it is working right now in development with this exact setting.

Reply:
That's what I like to hear!